internal/cmd/egctl/config_ratelimit.go

```diff
@@ -0,0 +1,206 @@
+// Copyright Envoy Gateway Authors
+// SPDX-License-Identifier: Apache-2.0
+// The full text of the Apache license is available in the LICENSE file at
+// the root of the repo.
+
+package egctl
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+
+	"github.com/envoyproxy/gateway/api/v1alpha1"
+	"github.com/envoyproxy/gateway/internal/envoygateway"
+	"github.com/envoyproxy/gateway/internal/infrastructure/kubernetes/ratelimit"
+	"github.com/envoyproxy/gateway/internal/kubernetes"
+	"github.com/spf13/cobra"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+	"k8s.io/apimachinery/pkg/types"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	cmdutil "k8s.io/kubectl/pkg/cmd/util"
+)
+
+var (
+	defaultRateLimitNamespace = "envoy-gateway-system" // TODO: make this configurable until EG support
+	defaultConfigMap          = "envoy-gateway-config" // TODO: make this configurable until EG support
+	defaultConfigMapKey       = "envoy-gateway.yaml"   // TODO: make this configurable until EG support
+)
+
+func ratelimitConfigCommand() *cobra.Command {
+
+	var (
+		namespace string
+	)
+
+	rlConfigCmd := &cobra.Command{
+		Use:     "envoy-ratelimit",
+		Aliases: []string{"rl"},
+		Long:    `Retrieve the relevant rate limit configuration from the Rate Limit instance`,
+		Example: `  # Retrieve rate limit configuration
+  egctl config ratelimit
```

Comment:
```suggestion
  egctl config envoy-ratelimit
```

Reply:
BTW, can you also update this command into https://gateway.envoyproxy.io/latest/design/egctl/#examples

Reply:
> BTW, can you also update this command into https://gateway.envoyproxy.io/latest/design/egctl/#examples

the `egctl config envoy-gateway` command is outdated, can you help remove it? thanks in advance